tests/rgen/Microsoft.Macios.Generator.Tests/Emitters/BindingSyntaxFactoryArgumentsTests.cs

```diff
@@ -426,7 +426,7 @@ public void MyMethod (Callback cb) {}
 ";
 			yield return [
 				nullableStringParameter,
-				$"var nsstringParameter = {Global ("CoreFoundation")}.CFString.CreateNative (stringParameter);\n",
+				$"using var nsstringParameter = {Global ("CoreFoundation")}.CFString.CreateNative (stringParameter);\n",
```

Comment:
`CFString.CreateNative` returns a `NativeHandle`, which doesn't work with `using`.

Reply:
I missed that, fixing.